frontend/src/app/modules/availability/availability-page/availability-page.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'app-availability-page',
+    templateUrl: './availability-page.component.html',
+    styleUrls: ['./availability-page.component.sass'],
+})
+export class AvailabilityPageComponent implements OnInit {
+    // eslint-disable-next-line @angular-eslint/no-empty-lifecycle-method
```

Comment:
Please do not use eslint-disable

Reply:
Can I remove 'implement OnInit' here? Because I don't use it for now.

Reply:
Yes